src/z3c/recipe/tag/__init__.py

```diff
@@ -136,7 +136,7 @@ def _build_idutils(self):
             ], [
                 '-m',
                 pkg_resources.resource_filename(
-                    "z3c.recipe.tag", "id-lang.map"),
+                    "z3c.recipe.tag", "id-lang.map.txt"),
```

Comment:
This is an interesting change.

May I ask why?

Reply:
The reason is not to have a separate rule in `MANIFEST.in` for this one project. As the `MANIFEST.in` is in `meta/config/default` thus shared for all packages.